frontend/src/routes/AccessControlManagement/AccessControlManagementForm.tsx

```diff
@@ -263,9 +275,9 @@ const AccessControlManagementForm = ({
             placeholder: subjectType === 'Group' ? t('Select or enter group name') : t('Select or enter user name'),
             value: selectedUserNames,
             onChange: (values) => setSelectedUserNames(values),
-            options: (subjectType === 'Group' ? allGroups : allUsers).map((val) => ({
-              id: val,
-              value: val,
+            options: (subjectType === 'Group' ? groups ?? [] : users ?? []).map((val) => ({
```

Comment:
it is not a good pattern to nest ternary operator, see https://eslint.org/docs/latest/rules/no-nested-ternary
```suggestion
            options: ((subjectType === 'Group' ? groups : users) || []).map((val) => ({
```

Reply:
@kurwang did you notice https://redhat-internal.slack.com/archives/C08N0NBMM2S/p1747353094777809?